tests/test_utils/test_tensor_move.py

```diff
@@ -1,34 +1,54 @@
 import pytest
 
 from colossalai.utils.cuda import get_current_device
-from colossalai.utils.memory_utils.utils import colo_model_data_tensor_move, colo_model_data_tensor_move_inline
+from colossalai.utils.memory_utils.utils import colo_tensor_mem_usage, colo_model_data_tensor_move, colo_model_data_tensor_move_inline, colo_set_process_memory_fraction, colo_cuda_memory_capacity
 from colossalai.utils import free_port
-from colossalai.zero.sharded_param import ShardedTensor
+from colossalai.zero.sharded_param.tensorful_state import StatefulTensor
 import colossalai
 
 import torch
 
 from functools import partial
 import torch.multiprocessing as mp
 
+def _run_colo_tensor_mem_usage():
+    for t in [StatefulTensor(torch.randn(2, 3)), torch.randn(2, 3)]:
+        cuda_use, cpu_use = colo_tensor_mem_usage(t)
+        print("cuda_use")
+        print(cuda_use)
+        print("cput_use")
+        print(cpu_use)
+
+def _run_colo_set_process_memory_fraction():
+    colo_set_process_memory_fraction(0.5)
+
+def _run_colo_cuda_memory_capacity():
+    frac = colo_cuda_memory_capacity
+    print("cuda memory capacity of the current cuda")
```

Comment:
rm the print and check something useful.

Reply:
done